app/Rest/Employees_Controller.php

```diff
@@ -0,0 +1,475 @@
+				'relevance',
+				'slug',
+				'include_slugs',
+				'title',
+			),
+		);
+
+		return $query_params;
+	}
+
+	/**
+	 * Get_item_schema
+	 *
+	 * @return array
+	 */
+	public function get_item_schema() {
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => Employee_Post_Type::$ID,
+			'type'       => 'object',
+			'properties' => array(
+				'title'     => array(
+					'description' => sprintf( __( 'The title for the %s.' ), Employee_Post_Type::$ID ),
+					'type'        => 'string',
+				),
+				'content'   => array(
+					'description' => sprintf( __( 'The content for the %s.' ), Employee_Post_Type::$ID ),
+					'type'        => 'string',
+				),
+				'acf_field' => array(
+					'description' => sprintf( __( 'The ACF field for the %s.' ), Employee_Post_Type::$ID ),
+					'type'        => 'object',
+				),
+			),
+		);
+
+		return $this->add_additional_fields_schema( $schema );
+	}
+
+	/**
+	 * Prepare_item_for_database
+	 *
+	 * @param \WP_REST_Request $request Full details about the request.
+	 *
+	 * @return object|\stdClass|\WP_Error
+	 */
+	protected function prepare_item_for_database( $request ) {
+		$prepared_post = new \stdClass();
+
+		if ( isset( $request['id'] ) ) {
+			$existing_post = $this->get_employee( $request['id'] );
+			if ( is_wp_error( $existing_post ) ) {
+				return $existing_post;
+			}
+
+			$prepared_post->ID = $existing_post->ID;
+		}
+
+		$schema = $this->get_item_schema();
+
+		if ( ! empty( $schema['properties']['title'] ) && isset( $request['title'] ) ) {
+			if ( is_string( $request['title'] ) ) {
+				$prepared_post->post_title = $request['title'];
+			}
+		}
+
+		if ( ! empty( $schema['properties']['content'] ) && isset( $request['content'] ) ) {
+			if ( is_string( $request['content'] ) ) {
+				$prepared_post->post_content = $request['content'];
+			}
+		}
+
+		return $prepared_post;
+
+	}
+
+	/**
+	 * Update_acf_value
+	 *
+	 * @param array $args Arguments.
+	 * @param int   $post_id Post ID.
+	 *
+	 * @return \WP_Error|bool
+	 */
+	private function update_acf_value( $args, $post_id ) {
+		if ( empty( $args ) ) {
+			return new \WP_Error( 'rest_employee_empty_acf_object', __( 'Empty ACF fields object.' ), array( 'status' => 500 ) );
+		}
+
+		foreach ( $args as $selector => $value ) {
+			if ( empty( $value ) ) {
+				continue;
+			}
+
+			update_field( $selector, $value, $post_id );
+		}
+
+		return true;
+	}
+
+	/**
+	 * Get the Employee post, if the ID is valid.
+	 *
+	 * @param int $id Supplied ID.
+	 *
+	 * @return \WP_Post|\WP_Error Post object if ID is valid, WP_Error otherwise.
+	 */
+	private function get_employee( $id ) {
+		$error = new \WP_Error( 'rest_employee_invalid_id', __( 'Invalid employee ID.' ), array( 'status' => 404 ) );
+		if ( (int) $id <= 0 ) {
+			return $error;
+		}
+
+		$post = get_post( (int) $id );
+		if ( empty( $post ) || empty( $post->ID ) || Employee_Post_Type::$ID !== $post->post_type ) {
+			return $error;
+		}
```

Comment:
seems like duplicate a code to get employee from DB again.

you can call method you used before and just check for if (is_wp_error()) { return "my custom error for invalid ID";    }